operator/v1/types_ingress.go

```diff
@@ -99,6 +99,13 @@ type IngressControllerSpec struct {
 	// the generated certificate's CA will be automatically integrated with the
 	// cluster's trust store.
 	//
+	// If a wildcard certificate is used and shared by multiple HTTP/2
+	// enabled routes (which implies APLN) then clients (i.e., notably
+	// browsers) are at liberty to reuse open connections. This means a
+	// client can and will reuse a connection to another route and that is
```

Comment:
Why "and will"?  Isn't it up to the client's discretion?

Reply:
Dropped "and will".